src/views/SecurityAndAccess/Policies/Policies.vue

```diff
@@ -1,465 +1,448 @@
-                  <icon-time />
+                  <status-icon status="time" />
                 </info-tooltip>
               </dt>
               <dd>
                 {{ $t('pagePolicies.hostUsbDescription') }}
               </dd>
             </dl>
-            <b-form-checkbox
+            <BFormCheckbox
               id="hostUsbSwitch"
-              v-model="hostUsbState"
+              v-model="Policies.hostUsbEnabled"
               data-test-id="policies-toggle-hostUsb"
               switch
-              @change="changeHostUsbState"
+              @update:modelValue="changeHostUsbState"
             >
-              <span class="sr-only">
+              <span class="visually-hidden">
                 {{ $t('pagePolicies.hostUsb') }}
               </span>
-              <span v-if="hostUsbState">
+              <span v-if="Policies.hostUsbEnabled">
                 {{ $t('global.status.enabled') }}
               </span>
               <span v-else>{{ $t('global.status.disabled') }}</span>
-            </b-form-checkbox>
-          </b-col>
-        </b-row>
-        <b-row v-if="isAdminUser || isServiceUser" class="section-divider">
-          <b-col class="d-flex align-items-center justify-content-between">
+            </BFormCheckbox>
+          </BCol>
+        </BRow>
+        <BRow
+          v-if="username === 'admin' || username === 'service'"
+          class="section-divider"
+        >
+          <BCol class="d-flex align-items-center justify-content-between">
             <dl class="mt-3 mr-3 w-75">
               <dt>{{ $t('pagePolicies.acfUploadEnablement') }}</dt>
               <dd>
                 {{ $t('pagePolicies.acfUploadEnablementDescription') }}
               </dd>
             </dl>
-            <b-form-checkbox
+            <BFormCheckbox
               id="unauthenticatedACFUploadEnablementSwitch"
-              v-model="unauthenticatedACFUploadEnablementState"
+              v-model="Policies.unAuthenticatedACFUploadEnablementState"
               data-test-id="policies-toggle-unauthenticatedACFUploadEnablement"
               switch
-              @change="changeUnauthenticatedACFUploadEnablement"
+              @update:modelValue="changeUnauthenticatedACFUploadEnablement"
             >
-              <span class="sr-only">
+              <span class="visually-hidden">
                 {{ $t('pagePolicies.usbFirmwareUpdatePolicy') }}
               </span>
-              <span v-if="unauthenticatedACFUploadEnablementState">
+              <span v-if="Policies.unAuthenticatedACFUploadEnablementState">
                 {{ $t('global.status.enabled') }}
               </span>
               <span v-else>{{ $t('global.status.disabled') }}</span>
-            </b-form-checkbox>
-          </b-col>
-        </b-row>
-      </b-col>
-    </b-row>
-  </b-container>
+            </BFormCheckbox>
+          </BCol>
+        </BRow>
+      </BCol>
+    </BRow>
+    <BModal
```

Comment:
Try to implement in composables here. will be required for upcoming implementations.